spotbugs/src/main/java/edu/umd/cs/findbugs/Version.java

```diff
@@ -90,7 +96,7 @@ public static void registerApplication(String name, String version) {
         return applicationVersion;
     }
 
-    public static void main(String[] argv) throws InterruptedException {
+    public static void main(String[] argv) {
```

Comment:
I'm not sure if main() the best place to introduce logging, as usually main() is called from the CLI and the output is (usually) expected to go to the STD OUT/ERR. Just assuming someone manages to redirect Java logging to a file, executing it will just don't do anything visible.

For the rest of the source code logging makes sense of course.

Reply:
If logging is redirected to a file, then I would assume that that's the desired behavior, so I would hesitate to basically override it by forcing STDOUT here.